README.md

```diff
@@ -258,7 +258,7 @@ TrackHttpFailureRequest.Builder()
     .Message(message)
     .NetworkSpeed(networkSpeed)
     .Create(
-    trackHttpFailureRequest => MikrosManager.Instance.AnalyticsController.LogPresetInternalEvents(trackHttpFailureRequest),
+    trackHttpFailureRequest => MikrosManager.Instance.AnalyticsController.LogEvent(trackHttpFailureRequest),
```

Comment:
👍

Reply:
@drxeno02 
Should i approve this PR, or you will approve this.